### ISSUE TITLE
install nvidia libs like HOW?

### ISSUE THREAD

**Issue by ObsidianStack** [snowfruit/Daz-3D#1]
About your "guide":

What bottles runner? soda? wine? 
What preset? App \ gaming \ Custom? 

Then Install "setup_nvlibs.sh".
No WINEPREFIX? simply ./setup_nvlibs.sh ?

when writing a guide **be detailed**

No matter what runner, it returns: 

00dc:err:wgl:X11DRV_WineGL_InitOpenglInfo  couldn't initialize OpenGL, expect problems

Driver nvidia installed, 
D.E. : XFCE4
Bottles installed through Flatpak.

**Comment by snowfruit** [comment]
I agree. The guide needs lots of work. Right now it's here to help me remember how I did it "last time it worked". Check out the daz forums for more help and info.
